benchmarks/KWay/MergeAlg.hs

```diff
@@ -0,0 +1,36 @@
+{-# language BangPatterns #-}
```

Comment:
I know that pragmas are case-insensitive, but I'd prefer a consistent approach (we're using `LANGUAGE` everywhere else).

Reply:
I'd prefer to make all the `language` pragmas lower case. I am absolutely sick of unnecessary shift key holding, and I don't have a capslock.

Reply:
Ok, fine.